sim/paladin/holy_shock.go

```diff
@@ -9,9 +9,18 @@ import (
 )
 
 func (paladin *Paladin) registerHolyShock() {
+
+	hasInfusionOfLight := paladin.hasRune(proto.PaladinRune_RuneWaistInfusionOfLight)
+	var cdTime time.Duration
+	cdTime = 30
+	if hasInfusionOfLight {
+		cdTime = 6
+	}
+
+
 	paladin.holyShockCooldown = &core.Cooldown{
 		Timer:    paladin.NewTimer(),
-		Duration: time.Second * 15,
+		Duration: time.Second * cdTime,
```

Comment:
Instead of separating the number and unit can you add the unit to the variable itself?
```
cdTime := time.Second * 30
if hasInfusionOfLight {
  cdTime := time.Second * 6
}
```